Cargo.toml

```diff
@@ -20,7 +20,7 @@ members = [
   "sdk/template/azure_template_core",
   "sdk/template/azure_template",
   "sdk/storage/azure_storage_common",
-  "sdk/storage/azure_storage_blob",
+  "sdk/storage/azure_storage_blob", "sdk/cosmos/cosmosclient",
```

Comment:
Separate line.

Reply:
Cargo new just sticks it on the end. Disappointing. Will fix!

Reply:
Make sure you install Even Better Toml and there are formatting instructions in the repo. Granted, you have to re-save it in VSCode.

Reply:
Yeah, it's the resaving that got me. I didn't even open `Cargo.toml` 😅 [](url)